doc/nrf/app_dev/device_guides/nrf54l/building_nrf54l.rst

```diff
@@ -0,0 +1,127 @@
+.. _building_nrf54l:
+
+Building and programming with nRF54L15 DK
+#########################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This guide provides instructions on how to build and program the nRF54L15 development kit.
+Whether you are working with single or multi-image builds, the following sections will guide you through the necessary steps.
+
+Depending on the sample, you must program only the application core or both the Fast Lightweight Peripheral Processor (FLPR) and the application core.
+Additionally, the process will differ based on whether you are working with a single-image or multi-image build.
+
+Building for the application core
+*********************************
+
+For instructions on building for the application core only, see how to :ref:`program an application <programming_cmd>`.
+
+Building for the application and FLPR core
+******************************************
+
+This section outlines how to build and program for both the application and FLPR core, covering separate builds and sysbuild configurations.
+FLPR core supports two variants:
+
+* ``nrf54l15dk/nrf54l15/cpuflpr`` where FLPR runs from SRAM (recommended).
+   To build FLPR image with this variant, the application core image must include the ``nordic-flpr`` snippet.
+
+* ``nrf54l15dk/nrf54l15/cpuflpr/xip`` where FLPR runs from RRAM.
+   To build FLPR image with this variant, the application core image must include the ``nordic-flpr-xip`` snippet.
+
+Separate images
+---------------
+
+To build and program the application sample and the FLPR sample as separate images, follow these steps:
+
+.. tabs::
+
+   .. group-tab:: west
+
+      1. |open_terminal_window_with_environment|
+      #. Build the application core image, and based on your build target include the appropriate snippet:
+
+         .. code-block:: console
+
+            west build -p -b nrf54l15dk/nrf54l15/cpuapp -S nordic-flpr --no-sysbuild
+
+      #. Flash the application core image by running the following command:
+
+         .. code-block:: console
+
+            west flash
+
+      #. Build the FLPR core image:
+
+         .. code-block:: console
+
+            west build -p -b nrf54l15dk/nrf54l15/cpuflpr --no-sysbuild
+
+      #. Once you have successfully built the FLPR core image, flash it onto your device:
+
+         .. code-block:: console
+
+            west flash
+
+   .. group-tab:: nRF Util
```

Comment:
nRF Util is just a utility here, not the default development environment. VSC extension also uses nRF Util. This whole section should be part of nRF Util documentation. https://docs.nordicsemi.com/bundle/nrfutil/page/nrfutil-device/guides/programming.html

I'll gladly help you coordinate the update of nRF Util docs, so that this PR and the update there go live together.